packages/client/src/models/mentor.js

```diff
@@ -20,39 +29,110 @@ const MentorConverter = {
         return new Mentor(
             data.name,
             data.email,
+            data.phone,
+            data.pronouns,
+            data.college,
+            data.avatar,
+            data.bio,
+            data.major,
             data.timezone,
+            // what is this for?
             data.about,
             data.subjects,
-            data.tags
+            data.tags,
+            data.gradeLevels
         );
     }
 };
 
+const phoneRegex = RegExp(
+    /^\(?([0-9]{3})\)?[-. ]?([0-9]{3})[-. ]?([0-9]{4})$/
+);
+
+const mentorSchema = yup.object().shape({
+    email: yup
+        .string()
+        .email()
+        .required('Email Required'),
+    name: yup
+        .string()
```

Comment:
Check for the format of the email here too.

Reply:
`.email()` does that.

Reply:
Woops. Missed that line.